llvm/test/Transforms/Yk/BasicBlockTracer.ll

```diff
@@ -0,0 +1,11 @@
+; RUNNING TEST EXAMPLE: llvm-lit llvm/test/Transforms/Yk/BasicBlockTracer.ll
+; RUN: llc -stop-after yk-basicblock-tracer-pass --yk-basicblock-tracer < %s  | FileCheck %s
+
+; CHECK-LABEL: define i32 @main()
```

Comment:
What would be the syntax to assert that the function call is injected at the start of the block?

Reply:
Added 👉 75c19896b70e269ffe9f671e7eb03c564003a2f1